servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/TrafficResilienceHttpServiceFilter.java

```diff
@@ -35,25 +33,20 @@
 import io.servicetalk.http.utils.TimeoutHttpServiceFilter;
 import io.servicetalk.transport.api.ServerListenContext;
 
-import java.time.Duration;
 import java.util.concurrent.TimeoutException;
 import java.util.function.BiConsumer;
-import java.util.function.BiFunction;
 import java.util.function.Consumer;
 import java.util.function.Function;
 import java.util.function.Supplier;
 import javax.annotation.Nullable;
 
-import static io.servicetalk.buffer.api.CharSequences.newAsciiString;
-import static io.servicetalk.concurrent.api.Single.succeeded;
-import static io.servicetalk.http.api.HttpHeaderNames.RETRY_AFTER;
+import static io.servicetalk.traffic.resilience.http.RejectionPolicy.defaultRejectionResponsePolicy;
```

Comment:
Consider adding a reference to this static method in `Builder.onRejectionPolicy` javadoc. Otherwise, it's hard to discover it even exists.